components/KiboAppHeader.vue

```diff
@@ -0,0 +1,337 @@
+            <SfIcon class="sf-header__icon">
+              <font-awesome-icon icon="map-marker-alt" class="fa-icon" />
+            </SfIcon>
+            <div class="kibo-header__icon">
+              <span class="kibo-header__icon-name"> Location</span>
+            </div>
+          </SfButton>
+          <SfButton
+            v-e2e="'app-header-account'"
+            class="sf-button--pure sf-header__action"
+            @click="handleAccountClick"
+          >
+            <SfIcon class="sf-header__icon">
+              <font-awesome-icon :icon="[accountIcon, 'user-circle']" class="fa-icon" />
+              <i class="far fa-user-circle"></i>
+            </SfIcon>
+            <div class="kibo-header__icon">
+              <span class="kibo-header__icon-name"> My Account</span>
+            </div>
+          </SfButton>
+          <SfButton v-e2e="'app-header-cart'" class="sf-button--pure sf-header__action">
+            <SfIcon class="sf-header__icon">
+              <font-awesome-icon icon="shopping-cart" class="fa-icon" />
+            </SfIcon>
+            <div class="kibo-header__icon">
+              <span class="kibo-header__icon-name"> Cart</span>
+            </div>
+          </SfButton>
+        </div>
+      </div>
+
+      <div class="line-2"></div>
+
+      <div class="kibo-mega-menu"></div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { SfImage, SfIcon, SfButton, SfSearchBar, SfMenuItem, SfLink } from "@storefront-ui/vue"
+import { computed, ref, onBeforeUnmount, defineComponent, watch } from "@vue/composition-api"
+import { clickOutside } from "@storefront-ui/vue/src/utilities/directives/click-outside/click-outside-directive.js"
+import {
+  mapMobileObserver,
+  unMapMobileObserver,
+} from "@storefront-ui/vue/src/utilities/mobile-observer.js"
+import debounce from "lodash.debounce"
+
+import { useAsync } from "@nuxtjs/composition-api"
+import useUiState from "@/composables/useUiState"
+import * as logo from "@/assets/kibo_logo.png"
+import { useUiHelpers } from "@/composables"
+import { useUser } from "@/composables/useUser"
+import { useSearchSuggestions } from "@/composables/useSearchSuggestions"
+import { userGetters } from "@/composables/getters"
+import { useNuxtApp } from "#app"
+
+export default defineComponent({
+  components: {
+    SfImage,
+    SfIcon,
+    SfButton,
+    SfSearchBar,
+    SfMenuItem,
+    SfLink,
+  },
+  directives: { clickOutside },
+  setup() {
+    const { setTermForUrl, getFacetsFromURL, getCatLink } = useUiHelpers()
+    const { result, search, loading } = useSearchSuggestions()
+    const nuxt = useNuxtApp()
+    const app = nuxt.nuxt2Context.app
+    const { user, load: loadUser } = useUser()
+    const searchValue = ref("")
+    const isAuthenticated = computed(() => {
+      return userGetters.isLoggedInUser(user.value)
+    })
+    const term = ref(getFacetsFromURL().phrase)
```

Comment:
for search suggestion, we don't need to do anything with the URL. 
Query Term should only be from the search bar input as user types

Reply:
query term from URL removed.